hat/conf/logback.xml

```diff
@@ -105,6 +105,14 @@
         <appender-ref ref="ASYNCCONSOLE" />
     </logger>
 
+    <logger name="org.hatdex.hat.she.service.InstagramMediaMapper" level="DEBUG" additivity="false">
```

Comment:
So that bit I dunno yet. I am aware of the different levels of log. In my head DEBUG makes no sense to be in the live deployment. MAYBE in hubats. But the logback files have so many debugs..